### ISSUE TITLE
no page numbering

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#858]
in this Neotropical Ichthyology there is no page numbering

![image](https://user-images.githubusercontent.com/4609956/72625761-ae6d0e80-3949-11ea-8f62-83138ea03990.png)

and in the metdata, there is only an enumber instead of the pagination given

![image](https://user-images.githubusercontent.com/4609956/72625838-d8263580-3949-11ea-89af-53b12d92bd5c.png)

this is an issue that will become more and more common with epublishing. Only we taxonomists insist on page numbers...

**Comment by myrmoteras** [comment]
https://zenodo.org/record/3611386

**Comment by gsautter** [comment]
What's the issue here? We've seen e-pagenumbers before, and metadata validation accepts them just fine ...

**Comment by myrmoteras** [comment]
there is not page numbers givben, nor is there a page number on any side. In this case then just enter the enumber as page number?

**Comment by gsautter** [comment]
Yes, that's what I meant ... simply enter the e-pagenumber as the pagination.
